inspire_utils/name.py

```diff
@@ -213,12 +213,12 @@ def pprint(self, initials_only=False):
         last_name = self.last
         suffixes = ', ' + self.suffix if self.suffix else ''
 
-        if initials_only:
+        if initials_only and last_name is not u'' and len(self.first_list) is not 1:
```

Comment:
you shouldn't use `is` for these things, and I don't think the second condition is needed. If there are no last names, we want to display the whole name.